Game/src/Scenes/Gen_Level.js

```diff
@@ -59,13 +59,14 @@ class Gen_Level extends Phaser.Scene {
         //let thisPlayer = this.players.find(player => player.serverId === id);
 
         //Initialize physics
-        let finishLineOverlap = this.physics.add.overlap(thisPlayer, this.backgroundLayer.finishLine, this.winCallback, null, this); //[HERE] it does not work!
         this.physics.add.collider(thisPlayer, this.wallsLayer, null, null, this);
-        this.physics.add.collider(thisPlayer, this.obstaclesLayer, this.takeDamageCallback, null, this);
+        this.obstaclesLayerCollision = this.physics.add.collider(thisPlayer, this.obstaclesLayer, this.takeDamageCallback, null, this);
         this.physics.add.overlap(thisPlayer, this.powerUpBoxes, this.pickPowerUpCallback, null, this);
+        this.physics.add.overlap(thisPlayer, this.finishLine, this.winCallback, null, this);
+
 
-        console.log(this.backgroundLayer.finishLine);   //[HERE] The finishLine is undefined
-        console.log(finishLineOverlap); //[HERE] As you can see, the object2 is undefined
```

Comment:
Comment these two lines in the master code

Reply:
Both lines have been commented after #31